modules/service/src/main/scala/lucuma/itc/service/ItcMappings.scala

```diff
@@ -589,7 +589,42 @@ object ItcMapping extends Encoders {
                       case _ => "Error parsing point profile".leftIorNec
                     }
                     s
-                  case _ => "Error parsing point profile".leftIorNec
+                  case _ => "Emission lines are not supported yet".leftIorNec
+                }
+                point.map(p => cursorEnvAdd("sourceProfile", p)(i)).leftProblems.flatten
+              case ("point", AbsentValue) ::
+                  ("uniform", AbsentValue) ::
+                  ("gaussian", ObjectValue(ov)) :: Nil =>
+                val point: IorNec[String, SourceProfile] = ov match {
```

Comment:
What is happening here?  This seems to be the gaussian case so i expected to see `gaussian` here instead of `point`.

Reply:
Oh yes it is a naming thing. let me fix it